homeassistant/components/pglab/entity.py

```diff
@@ -0,0 +1,59 @@
+"""Entity for PG LAB Electronics."""
+
+from __future__ import annotations
+
+from pypglab.device import Device
+from pypglab.entity import Entity
+
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC, DeviceInfo
+from homeassistant.helpers.entity import Entity as HA_Entity
+
+from .const import DEVICE_ALREADY_DISCOVERED, DOMAIN
+
+
+class BaseEntity(HA_Entity):
```

Comment:
New integration must use `_attr_has_entity_name = True`

Reply:
https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations

Reply:
OK